node_monitor/node_monitor_helpers/sql_constants.py

```diff
@@ -0,0 +1,22 @@
+### SQL queries for table `subscribers`
+INSERT_SUBSCRIBER = """
+    INSERT INTO subscribers (
+        node_provider_id,
+        notify_on_status_change,
+        notify_email,
+        notify_slack,
+        node_provider_name,
+        notify_telegram
+    ) VALUES (%s, %s, %s, %s, %s, %s)
+    ON CONFLICT (node_provider_id) DO UPDATE SET
+        notify_on_status_change = EXCLUDED.notify_on_status_change,
+        notify_email = EXCLUDED.notify_email,
+        notify_slack = EXCLUDED.notify_slack,
+        node_provider_name = EXCLUDED.node_provider_name,
+        notify_telegram = EXCLUDED.notify_telegram
+"""
+
+DELETE_SUBSCRIBER = """
+    DELETE FROM subscribers
+    WHERE node_provider_id = %s
+"""
```

Comment:
Don't do this.
Just put these sql statements directly in the methods.
Also, since we're back to using node_provider_lookup, our sql statements will be significantly shorter. Also, you should delete the on conflict clause, there should not be any conflicts if the logic in the rest of the program is correct. Throwing an error in this case would be good.